signer/src/proto/convert.rs

```diff
@@ -1791,4 +2103,54 @@ mod tests {
         let original_from_proto = T::try_from(proto_original).unwrap();
         assert_eq!(original, original_from_proto);
     }
+
+    // The following are tests for structs that do not derive eq
+    #[derive(Debug)]
+    struct PartyStateWrapper((u32, PartyState));
+
+    impl PartialEq for PartyStateWrapper {
+        fn eq(&self, other: &Self) -> bool {
+            self.0 .0 == other.0 .0
+                && self.0 .1.nonce == other.0 .1.nonce
+                && self.0 .1.polynomial == other.0 .1.polynomial
+                && self.0 .1.private_keys == other.0 .1.private_keys
+        }
+    }
+
+    #[derive(Debug)]
+    struct SignerStateWrapper(SignerState);
+
+    impl PartialEq for SignerStateWrapper {
```

Comment:
@xoloki can we implement `PartialEq` for `PartyState` and `SignerState`? Any reason not to?

Reply:
Done @djordon